iterative/resource_task.go

```diff
@@ -173,6 +173,7 @@ func resourceTaskCreate(ctx context.Context, d *schema.ResourceData, m interface
 
 	task, err := resourceTaskBuild(ctx, d, m)
 	if err != nil {
+		utils.SendJitsuEvent("task_create", err, d)
```

Comment:
Correct me if I'm wrong, but aren't these all blocking operations, I recall reading that they should not?
```suggestion
		go utils.SendJitsuEvent("task_create", err, d)
```

Reply:
I do not think that its a huge deal since we are just sending data like calling Cloud APIs, I would not say that calling it once per creation/destroy is a lot. But I will check it

Reply:
Why?

Reply:
Fallacy number 1: _the network is reliable_ :wink:

Reply:
> Fallacy number 1: the network is reliable

Wont fail everything including the Vendors?

> If our telemetry server goes down, we don't want our product to fail

It works that way right now. I tested that to be sure that if the telemetry is not up we are not breaking the code

Reply:
> _Wont fail everything including the Vendors?_

If our telemetry server goes down, cloud providers may still work.

Reply:
> _If the telemetry is not up we are not breaking the code_

Connections can hang (again, network is unreliably), and `net/http` doesn't have any timeout by default. ✈️